singularity/Singularity.def

```diff
@@ -0,0 +1,51 @@
+Bootstrap: docker
+From: condaforge/mambaforge
+Stage: build
+
+%files
+script.sh /scripts
+%labels
+MAINTAINER Reimar Bauer <rb.proj@gmail.com>
+%post
+##################################################################################
+# Definition File for singualarity
+#
+# Based on mambaforge Image
+# singularity build -f mss.sif Singularity.def
+# singularity shell mss.sif 
+#
+# Singularity > mss # starts the ui
+# Singularity > mswms_demodata --seed  # creates in your $HOME a mss/ folter with testdata
+# Singularity > export PYTHONPATH=$HOME/mss; mswms # starts the development server 
+# Singularity > mscolab db --init; mscolab start # starts the mscolab server
+#
+##################################################################################
+
+
+
+# get keyboard working for mss gui
+apt-get --yes update && DEBIAN_FRONTEND=noninteractive \
+apt-get --yes install xserver-xorg-video-dummy \
+&& apt-get --yes upgrade \
+&& apt-get clean \
+&& rm -rf /var/lib/apt/lists/*
+
+# Set up conda-forge channel
+conda config --add channels conda-forge && conda config --add channels defaults &&\
```

Comment:
doesnt that put defaults on top of forge, and didn't we stop doing that?

Reply:
ah, good catch.

Reply:
no it doesn't it appends, only when it is in the list already it would go on top.
Because of mamba-forge, we have already conda-forge set and that is not needed.

But we need defaults for the next command. No better solution yet.